src/Microsoft.Health.Dicom.Functions.Client/TaskHub/TrackingTable.cs

```diff
@@ -0,0 +1,42 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Data.Tables;
+using EnsureThat;
+
+namespace Microsoft.Health.Dicom.Functions.Client.TaskHub;
+
+internal abstract class TrackingTable
```

Comment:
Oh, this is what these tables are called in the Durable Task Framework: the "tracking tables." However, you're absolutely right -- this could apply to any table.